target/linux/ath79/dts/qca9531_glinet_gl-e750.dts

```diff
@@ -145,6 +158,6 @@
 &wmac {
 	status = "okay";
 
-	nvmem-cells = <&cal_art_1000>;
-	nvmem-cell-names = "calibration";
+	nvmem-cells = <&cal_art_1000>, <&macaddr_art_1002>;
+	nvmem-cell-names = "calibration", "mac-address";
```

Comment:
`art@1002` is the default value, we don't need to touch it.

Reply:
here and elsewhere?

Reply:
I just checked here.

Reply:
Right. Different PR.